doc/source/diff_v3_v4.rst

```diff
@@ -499,7 +499,7 @@ Modules Specific Tcl Commands
 
 **is-loaded**
 
- Starting with version ``4.1``, **is-loaded** supports being called with no argument passed. In this case, it returns *true* if any modulefile is currently loaded, *false* elsewhere.
```

Comment:
More semantic markup can be done here and in other files, but that will come later if this PR is accepted.